### PR TITLE
docs: info for more editors FUI-1591

### PR DESCRIPTION
**Pull request by matteematt**
🤔  &nbsp; **What does this PR do?**

- Fixes the path of the gifs about vim, which were missed in #76 
- Adds a table of info about supported editors to the main readme and the lsp readme
- Adds info about JetBrains in the LSP readme

📑  &nbsp; **How should this be tested?**

Only docs changes

✅  &nbsp; **Checklist**

- [ ] I have added tests for my changes. - N/A
- [X] I have updated the project documentation.
- [X] I have followed the [contribution guidelines](https://github.com/genesiscommunitysuccess/custom-elements-lsp/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Review by MrBrunoWolff — APPROVED** [review]

File: packages/core/custom-elements-lsp/README.md [hunk 1]
```diff
@@ -129,6 +137,19 @@ You just need to setup VSCode to use your local typescript install as by default
...
+This section covers all of JetBrains IDEs, such as WebStorm and IntelliJ.
+[Currently](https://youtrack.jetbrains.com/issue/WEB-62815/Ability-to-use-tsserver-to-implement-all-LSP-functionality-from-TypeScript) there is only partial support for the CEP.
+* Full diagnostics support.
+* Partial completions support, provided by the IDE itself *not* using the CEP - but enhanced because of the manifest files that are used with the plugin.
+* No support for quicklook/quickinfo, as JetBrains run this closed source without using `tsserver` in their IDEs.
+
+To use the LSP in your JetBrains IDE.
+1. Launch the IDE with the project at the root of the monorepo.
+2. Open the preferences menu option from the settings.
+3. Navigate to the `Typescript` settings in the `Languages & Frameworks` settings, and ensure that the typescript option is set to the `node_modules/typescript` of your _local_ project, as shown in the image. This may be the default already, in which case you don't need to do anything. You'll also want to enable at least the three options which are enabled in the image below.
```
Comment (derekdon) [inline packages/core/custom-elements-lsp/README.md:151]: > which are enabled in the image below.

what image? do you mean above?

File: README.md [hunk 2]
```diff
@@ -1,8 +1,16 @@
+The [Genesis Global](https://genesis.global) Community Success initiative is committed to open-sourcing select technologies that we believe the open-source community would benefit from.
+
 # Custom Elements Monorepo
 
 This repository contains multiple packages which can be used to work with [Custom Elements](https://developer.mozilla.org/en-US/docs/Web/API/Web_components/Using_custom_elements).
 
-To get started run `$ npm run bootstrap` in your shell.
+| Editor | Completions | Diagnostics | Quickinfo | Info |
```
Comment (derekdon) [inline README.md:7]: Yep, looks much better!

**Review by derekdon — APPROVED** [review]
LGTM, minor comment about image placement.

File: packages/core/custom-elements-lsp/README.md (hunk 1 above)
Comment (matteematt) [reply packages/core/custom-elements-lsp/README.md:175]: Thanks, addressed in https://github.com/genesiscommunitysuccess/custom-elements-lsp/pull/80/commits/593fcd3341c369e5a8294c9852cff9cca1863480

**Review by derekdon — APPROVED** [review]

**Pull request merged**